mock/service/service.go

```diff
@@ -73,6 +119,33 @@ func New(config Config) Service {
 		nodeID: config.DriverName,
 		config: config,
 	}
+	if config.ExecHooks != nil {
+		s.hooksVm = otto.New()
+		s.hooksVm.Run(`__grpcCode = {
+			OK: 0,
+			CANCELED: 1,
+			UNKNOWN: 2,
+			INVALIDARGUMENT: 3,
+			DEADLINEEXCEEDED: 4,
+			NOTFOUND: 5,
+			ALREADYEXISTS: 6,
+			PERMISSIONDENIED: 7,
+			RESOURCEEXHAUSTED: 8,
+			FAILEDPRECONDITION: 9,
+			ABORTED: 10,
+			OUTOFRANGE: 11,
+			UNIMPLEMENTED: 12,
+			INTERNAL: 13,
+			UNAVAILABLE: 14,
+			DATALOSS: 15,
+			UNAUTHENTICATED: 16
+		};`)
```

Comment:
It would be useful to have this in some `globals.js` file in this repo, so it can be linked from documentation easily && edited without go knowledge.

The file itself can be either embedded into csi-mock binary via bindata or it can be shipped in the container with the driver.

Reply:
Or, create `hook_globals.go` with a single string constant and link it from docs...

Reply:
Also -- let's remove the `__grpc` object and leave just the constants there. This behaves unexpectedly when one makes a typo (no error encountered).